pkg/metrics/metrics.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
So the sidecars use csi-lib-utils' MetricsManager, although I don't think that's appropriate here.

csi-lib-utils itself uses k8s.io/component-base/metrics/.

I have no idea if there's an advantage or good reason to using that rather than prometheus directly.

@verult ?

Reply:
I'm actually not sure. @Jiawei0227 do you happen to know?

Reply:
Saikat and I discussed offline and we think k8s.io/component-base/metrics is the right thing to use. ???

Reply:
We should use `k8s.io/component-base/metrics` as it is the moving standard kubernetes metrics wrapper pkg

Reply:
We should not use MetricsManager in csi-lib-utils I think. That might not be a good fit

Reply:
@Jiawei0227  I am planning to use k8s.io/component-base/metrics directly, not the csi lib utils.